inc/Game.hpp

```diff
@@ -1,15 +1,14 @@
 #pragma once
 #include <string>
+#include <vector>
 
 class Game
 {
     // TODO: Add game status: No game, In progress, Finished
 public:
     Game();
-    void addFrames(const std::string & frames);
```

Comment:
Usunąłeś (chyba przypadkowo) funkcje addFrames i read podczas zastępowania plików. Wrzuć je jeszcze z powrotem i wszystko będzie hulać. Wiem, że miały one pustą implementację i niczego w takiej postaci nie wnosiły, ale ich istnienie to był pierwszy krok do innych zadań, związanych z wczytywaniem danych z pliku.

Reply:
rzeczywiście, już dodałem   `void Game::addFrames(std::string const & frames) {} ` do Game.cpp 
i prototyp do Game.hpp